spec/lib/percy/capybara/loaders/base_loader_spec.rb

```diff
@@ -14,6 +14,18 @@
       expect(resource.content).to include('Hello World!')
       expect(resource.sha).to eq(Digest::SHA256.hexdigest(resource.content))
     end
+    it 'includes an iframe resource' do
```

Comment:
I do need to put that test on a proper level/context,
however if I write it without `js: true`, for consistency I'd have to also do this for https://github.com/percy/percy-capybara/blob/master/spec/lib/percy/capybara/loaders/base_loader_spec.rb#L4,
and write an sample Rack app to answer requests like https://github.com/percy/percy-capybara/blob/master/spec/lib/percy/capybara/loaders/sprockets_loader_spec.rb#L3
Should I actually do this or should I just keep the `js: true` like the `#root_html_resource` test?